openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/travail_prive.py

```diff
@@ -804,7 +804,7 @@ def formula(individu, period, parameters):
 class mmid_salarie(Variable):
     value_type = float
     entity = Individu
-    label = 'Cotisation maladie (salarié)'
+    label = 'Cotisation salariale supplémentaire Alsace Moselle'
```

Comment:
Je suis un peu gêné par ce changement de label qui n'est exact qu'après 2018 parce que seule l'Alsace-Moselle conserve une cotisation maladie.  Est-ce que l'on ne pourrait pas tenter une label mixte ?

Ou sinon créer un variable Alsace-Moselle spécifique ?

Reply:
Je suis gênée aussi ... en fait j'ai effectué surtout le changement pour qu'on en discute. Dans LexImpact je peux modifier le label. Donc on pourrait garder le label actuel dans OpenFisca. Qu'en penses-tu ?

Reply:
Cela me paraît une bonne solution pour merger cela rapidement à moindre coût.

Reply:
Ok je retire ma modification ;) 
```suggestion
    label = 'Cotisation maladie (salarié)'
```